draft-gpew-priv-ppm.md

```diff
@@ -586,6 +587,12 @@ type 'outdatedConfig'. Clients SHOULD invalidate any cached aggregator
 `HpkeConfig` and retry with a freshly generated Report. If this retried report
 does not succeed, clients MUST abort and discontinue retrying.
 
+The leader MUST ignore any report whose nonce contains a timestamp that falls in
+a batch interval for which it has received at least one collect request from the
+collector. (See {{pa-collect}}.) Otherwise, comparing the aggregate result to
```

Comment:
This seems like a recipe for silent failure in case of clock skew.

Reply:
Added text saying the leader SHOULD reply with an error. As I noted in a previous thread, it may not be feasible for the leader to decide if the batch interval has been collected. It's much simpler if all the server has to do is ingest the report at this step and wait to prune it when it's ready to start aggregating.